cql2pgjson/src/test/resources/compoundIndex.sql

```diff
@@ -18,7 +18,20 @@ $func$  LANGUAGE sql IMMUTABLE;
 create or replace function concat_space_sql(VARIADIC text[])
 RETURNS text AS $$ select concat_ws(' ', VARIADIC $1); $$ LANGUAGE SQL IMMUTABLE;
 
+create or replace function concat_array_object_values(jsonb_data text, field text) RETURNS text AS $$
+DECLARE
 
+  res text;
+  i jsonb;
+BEGIN
+  
+  FOR i IN SELECT * FROM json_array_elements(($1)::json)
```

Comment:
Can you use `jsonb_array_elements($1)`? This avoids converting the JSONB to a string and converting the string into a JSON.